### PR TITLE
Add Sitemaps to v5 upgrade guide

### PR DESCRIPTION
**Pull request by roborourke**

### CODE REVIEW

File: other-docs/guides/upgrading/v5.md [hunk 1]
```diff
@@ -53,6 +55,14 @@ If you're using Facebook Instant Articles, you will need to require the FBIA plu
 $ composer require wpackagist-plugin/fb-instant-articles
 ```
 
+#### Sitemaps
+
+The MSM Sitemaps plugin used to provide the XML Sitemaps feature however this has been superceded by the new core Sitemaps in WordPress 5.5. These were a collaboration between WordPress, Google and Human Made.
```
Comment (kadamwhite) [inline other-docs/guides/upgrading/v5.md:60]:
```suggestion
The MSM Sitemaps plugin previously used to provide the XML Sitemaps feature has been superseded by the new core support for Sitemaps introduced in WordPress 5.5. These core sitemaps were developed in collaboration between WordPress, Google and Human Made.
```

File: other-docs/guides/upgrading/v5.md [hunk 2]
```diff
@@ -53,6 +55,14 @@ If you're using Facebook Instant Articles, you will need to require the FBIA plu
...
+
+The update may require you to re-submit your sitemap index file in [Google Search Console](https://search.google.com/search-console/) as it has been relocated from `/sitemap.xml` to `/wp-sitemap.xml`. The new index sitemap URL is automatically added to the `robots.txt` file.
```
Comment (kadamwhite) [inline other-docs/guides/upgrading/v5.md:62]: Will the old sitemap URL definitely be no longer present? Do they need to keep it around in `robots.txt` if they had done any customization previously?

**Review by svandragt — APPROVED** [review]
All good. You can change the endpoint with a filter as well iirc.

**Review by kadamwhite — APPROVED** [review]
Left a re-wording suggestion on how to introduce the new feature -- in general looks good, I think as a user if I had been customizing the sitemaps I'd want a little more hand-holding, but I don't know it's our responsibility in this document to provide anything more in depth than this, so I'm +1

File: other-docs/guides/upgrading/v5.md (hunk 2 above)
Comment (roborourke) [reply other-docs/guides/upgrading/v5.md:62]: Correct, if anyone was using the built in MSM one it was being added to `robots.txt` in the same way and was also generating it so it's a clean switch in that regard.

If there have been any customisations then those will need to be ported across.

**Comment by roborourke** [comment]
Merging for the sake of getting beta 2 out but we can revisit with some specifics depending on feedback - fwiw I've documented the backend API on the seo module so there's more or less a 1:1 matching of how to do the same modifications.

**Pull request merged**